question-answering-using-openai/main.bal

```diff
@@ -0,0 +1,211 @@
+import ballerina/http;
+import ballerinax/googleapis.sheets;
+import ballerinax/openai;
+import ballerina/lang.'float as langFloat;
+import ballerina/regex;
+import ballerina/log;
+import ballerina/os;
+
+
+// Google Sheets API client configuration parameters
+final string gsheetsClientID = os:getEnv("GSHEETS_CLIENT_ID");
+final string gsheetsClientSecret = os:getEnv("GSHEETS_CLIENT_SECRET");
+final string gsheetsRefreshToken = os:getEnv("GSHEETS_REFRESH_TOKEN");
+configurable string sheetId = ?;
+configurable string sheetName = ?;
+
+// OpenAI token
+// configurable string openAIToken = ?;
+final string openAIToken = os:getEnv("OPENAI_KEY");
```

Comment:
Use configurables.

Reply:
addressed in [a3d99a2](https://github.com/ballerina-guides/ai-samples/pull/2/commits/a3d99a29fc85ec81cafb7f92df253d5cbe4b6449)